lfs/scanner.go

```diff
@@ -187,6 +195,12 @@ func revListShas(refLeft, refRight string, opt ScanRefsOptions) (chan string, er
 		if refRight != "" && !z40.MatchString(refRight) {
 			refArgs = append(refArgs, refRight)
 		}
+	case ScanAll:
+		refArgs = append(refArgs, "--all")
+	case ScanDeltaToRemotes:
+		refArgs = append(refArgs, refLeft, "--not", "--remotes")
```

Comment:
I'm using just "--remotes" here. Another option would be to use --remotes=<remote_name_from_pre-push_arg_1>. But since we only have one git-lfs server and single lfs.url configuration option, it is safe to assume that the same git lfs server is used for all remotes. So, we can do a better jobs isolating only lfs objects introduced locally.